CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ the detailed section referring to by linking pull requests or issues.
 
 #### Minor
 
+- Implemented markdown for asset descriptions
```

Comment:
![grafik](https://github.com/sovity/edc-ui/assets/30114507/bf01f740-5613-4f41-8b69-561026f7731b)

we should reduce the amount of font sizes that we use, note how the default prose text is a different weight (?) or at least size that we use in the rest of the application

compare (the no description text is not affected by the markdown rendering + tw typography):

![grafik](https://github.com/sovity/edc-ui/assets/30114507/efc03766-bda1-494a-9d7f-773a5569865d)